pkg/steps/template.go

```diff
@@ -836,3 +837,49 @@ func podLogNewContainers(podClient coreclientset.PodInterface, pod *coreapi.Pod,
 		notifier.Complete(pod.Name)
 	}
 }
+
+func printContainerLogsToStdout(podClient coreclientset.PodInterface, statusName, podName string) error {
+	if s, err := podClient.GetLogs(podName, &coreapi.PodLogOptions{
+		Container: statusName,
+	}).Stream(); err == nil {
+		if _, err := io.Copy(os.Stdout, s); err != nil {
+			return fmt.Errorf("error: Unable to copy log output from failed pod container %s: %v", statusName, err)
+		}
+		s.Close()
+	} else {
+		return fmt.Errorf("error: Unable to retrieve logs from failed pod container %s: %v", statusName, err)
+	}
+	return nil
+}
+
+func getContainersMap(containers string) map[string]struct{} {
+	s := strings.Split(containers, ",")
```

Comment:
I'm thinking of [stripping whitespace](https://golang.org/pkg/strings/#TrimSpace) to be friendly to people specifying the list as `aaa, bbb`